DetectorDescription/DDCMS/data/cms-geometry-2021.xml

```diff
@@ -46,8 +46,8 @@
     <Include ref='Geometry/CMSCommonData/data/cmsBeam/2026/v1/cmsBeam.xml'/>
     <Include ref='Geometry/CMSCommonData/data/muonMB.xml'/>
     <Include ref='Geometry/CMSCommonData/data/muonMagnet.xml'/>
-    <Include ref='Geometry/CMSCommonData/data/cavern/2018/v1/cavern.xml'/>
-    <Include ref='Geometry/CMSCommonData/data/cavernData/2017/v1/cavernData.xml'/>
+    <Include ref='Geometry/CMSCommonData/data/cavern/2018/v2/cavern.xml'/>
```

Comment:
should it be 2021/v1?

Reply:
Yes, it could be better. For both cavern files?